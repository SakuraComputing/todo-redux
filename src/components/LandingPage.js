import React from 'react';
import '../App.css';

const LandingPage = () => {
    return (
        <div className="container">
            <h1>TODO List</h1>
        </div>
    );
}
export default LandingPage;
