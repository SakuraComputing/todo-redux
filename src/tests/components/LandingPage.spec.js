import { shallow } from 'enzyme';
import React from 'react';
import LandingPage from '../../components/LandingPage';

describe('landing page component', () => {

    let wrapper;

    beforeEach(() => {
        wrapper = shallow(<LandingPage />);
    });

    it('should render correctly', () => {
        expect(wrapper).toMatchSnapshot();
    });
    it('should display h1 text', () => {
        const headerText = wrapper.find('h1').text();
        expect(headerText).toEqual('Late Rooms');
    })
});