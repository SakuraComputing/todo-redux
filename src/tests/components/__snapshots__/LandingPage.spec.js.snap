// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`landing page component should render correctly 1`] = `undefined`;
